sidechain/peer-fetch/src/mocks/fetch_blocks_from_peer_mock.rs

```diff
@@ -0,0 +1,60 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+use crate::{FetchBlocksFromPeer, Result};
+use async_trait::async_trait;
+use its_primitives::{
+	traits::SignedBlock as SignedBlockTrait,
+	types::{BlockHash, ShardIdentifier},
+};
+use std::collections::HashMap;
+
+pub struct FetchBlocksFromPeerMock<SignedBlock> {
+	signed_blocks_map: HashMap<ShardIdentifier, Vec<SignedBlock>>,
+}
```

Comment:
Added a new mock for the `FetchBlocksFromPeer` trait, to be used in unit tests in other crates (and hence also added a new `mocks` feature to this crate).